recipes/dcm2niix/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "dcm2niix" %}
+{% set version = "1.0.20180328" %}
+{% set sha256 = "3a5c2e2636021e6cafecb213c6ca4b5226bfcdadf00a241fde45b58f513ea674" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url:  https://github.com/rordenlab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
Should this be skipped? Was just looking at the `bld.bat` above and it seemed mostly ok. :/